commerce/domain/inventory/cart/calculator/classes/InventoryCartCalculatorSampleTest.cls

```diff
@@ -0,0 +1,199 @@
+@isTest
+public class InventoryCartCalculatorSampleTest {
+    
+	private static final String CART_NAME = 'My Cart';
+    private static final String ACCOUNT_NAME = 'My Account';
+    private static final String WEBSTORE_NAME = 'My WebStore';
+    private static final String DELIVERYGROUP_NAME = 'My Delivery Group';
+    private static final String CART_ITEM1_NAME = 'My Cart Item 1';
+    private static final String CART_ITEM2_NAME = 'My Cart Item 2';
+    private static final String CART_ITEM3_NAME = 'My Cart Item 3';
+    private static final String SKU1_NAME = 'My SKU 1';
+    private static final String SKU2_NAME = 'My SKU 2';
+    private static final String SKU3_NAME = 'My SKU 3';
+
+    @IsTest
+    public static void testNoCVOCartStatus() {
+      
+         // Arrange
+         CartExtension.Cart cart = arrangeAndLoadCartWithSpecifiedStatusAndThreeItems(CartExtension.CartStatusEnum.ACTIVE);
+         InventoryCartCalculatorSample calculator = new InventoryCartCalculatorSample(new DefaultInventoryCartCalculatoMockExecutor());
+
+        Test.startTest();
+        calculator.calculate(new CartExtension.CartCalculateCalculatorRequest(cart, CartExtension.OptionalBuyerActionDetails.empty()));
+        Test.stopTest();
+
+        // Assert
+        assertNoCartValidationOutputs(cart, 0);
+
+        
+    }
+    
+    @IsTest
+    public static void testCleanCVOCartStatus() {
```

Comment:
This test is no different from the previous one? Did you mean to add CVO to the Cart before acting on it?